src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -586,6 +586,20 @@
 			"# Can outlaws roam freely on the towns they are outlawed in?",
 			"# If false, outlaws will be teleported away if they spend too long in the towns they are outlawed in.",
 			"# The time is set below in the outlaw_teleport_warmup."),
+	GTOWN_SETTINGS_ALLOW_OUTLAWS_TO_TELEPORT_OUT_OF_TOWN(
+			"global_town_settings.allow_outlaws_to_teleport_out_of_town",
+			"true",
+			"",
+			"# Can outlaws freely teleport out of the towns they are outlawed in?",
+			"# If false, outlaws cannot use commands to teleport out of town.",
+			"# If you want outlaws to not be able to use teleporting items as well, use allow_outlaws_use_teleport_items."),
+	GTOWN_SETTINGS_ALLOW_OUTLAWS_USE_TELEPORT_ITEMS(
+			"global_town_settings.town_allows_outlaw_teleport_items",
+			"true",
+			"",
+			"# If false, outlawed players in towns cannot use items that teleport, ie: Ender Pearls & Chorus Fruit.",
+			"# If this is true and allow_outlaws_to_teleport_out_of_town is false, outlaws are still barred from using other methods of teleporting.",

```

Comment:
Should add a comment saying that this requires the above allow_outlaws_to_teleport_out_of_town to be false.